sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -447,15 +437,22 @@ export type State = "notStarted" | "running" | "succeeded" | "failed" | "rejecte
 // @public
 export type StringIndexType = "TextElements_v8" | "UnicodeCodePoint" | "Utf16CodeUnit";
 
+// @public
+export interface TextAnalyticsActions {
```

Comment:
in .NET and Java we have `AnalyzeBatchActions`. 
I actually like JS name better. What do you think @mssfang

Reply:
Agree. We can change it to TextAnalyticsActions.